ui/src/components/SchemaForm/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { RJSFSchema } from "@rjsf/utils";
+import validator from "@rjsf/validator-ajv8";
+
+import { Form } from "./Theme";
+
+const SchemaForm = () => {
+  const schema: RJSFSchema = {
```

Comment:
Can you keep boilerplate outside of any basic components? I know it says (WIP), but the SchemaForm should take in props, right? So if you can make the basic components how they should be and leave any wip/boiler stuff in storybook stories or separate files that'll mean we are producing components that are ready to use as well as reduce the future PR sizes as you finalize the functionality 🙏🏻